projects/storefrontlib/src/lib/occ/docs/cms-component-implementation.md

```diff
@@ -89,3 +89,24 @@ ConfigModule.withConfig({
   }
 });
 ```
+
+# Control of Server Side Rendering
+
+Some of the CMS components might be intended not to render in the server for multiple reasons:
+
+- a CMS component requires personalised input and should not or can not be rendered without it
+- a CMS component is not required for SSR output and for performance reason will be removed from the rendering process
+- a CMS component interacts with external services (latency) and is not relevant for indexing and social sharing
+
+Every CMS component is enabled in SSR by default, but can be configured no to render in the server:
```

Comment:
Can we add this bit to the doc: 

While it is possible to add conditional logic in a component to render (parts of) the view in the SSR, we offer a configuration for components to make this more generic and avoid any specific login in components.